content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -208,6 +208,71 @@ Adjust the connection URL to match your setup - see [Microsoft SQL Server connec
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'cockroachdb']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Note that the default schema created by `prisma init` uses PostgreSQL as the `provider`. For CockroachDB, you need to edit the `datasource` block to use the `cockroachdb` provider instead:
+
+```prisma file=prisma/schema.prisma highlight=2;edit
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+You will also need to add the `cockroachdb` preview feature flag to the `generator` block of your `schema.prisma` file:
+
+```prisma file=schema.prisma highlight=3;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["cockroachdb"]
+}
+
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+The `url` is [set via an environment variable](/guides/development-environment/environment-variables) which is defined in `.env`. You now need to adjust the connection URL to point to your own database.
```

Comment:
Offtopic:
General comment that applies to all tutorials: Before this sentence would be a good place to add a structuring headline that lets the user know that everything that follows is now about the connection string or URL. (vs. choosing the connector in the first half of the document)

Reply:
Added this to list for https://github.com/prisma/docs/issues/2747